back/api/routers/crud.py

```
@@ -1,102 +1,102 @@
+    return status_read_books
+
+# 積読本更新順
+@router.get("/store", response_model=List[books.Book])
+def read_books_store(skip: int = 0, limit: int = 100, session: Session = Depends(connection.get_db)):
+    status_store_books = session.query(book_model.Books).filter(book_model.Books.status == books.BookStatus.STORE.value).order_by(book_model.Books.last_modified).offset(skip).limit(limit).all()
+    return status_store_books
+
+
+# 特定の書籍取得
+@router.get("/{book_id}", response_model=books.Book)
+def read_book(book_id: int, session: Session = Depends(connection.get_db)):
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)